nemo/collections/nlp/data/language_modeling/l2r_lm_dataset.py

```diff
@@ -42,11 +47,12 @@ def __init__(
         dataset: str,
         max_seq_length: Optional[int] = 512,
         batch_step: Optional[int] = None,
+        cache_ids: bool = False,
```

Comment:
Could you please update the docstring?

Reply:
We used the term use_cache instead of cache_id in nlp collection. Do you think it is OK to change the name to have consistency?

Reply:
Done